### ISSUE TITLE
2019.12 Copy and paste issue

### ISSUE THREAD

**Issue by Poporacer** [smeighan/xLights#1480]
2019.12 has a problem with the copy and paste in the Lyric Tracks. (I don't know if it is elsewhere) If you copy the words and the broke down words, when you paste the selection, you have to select 3 rows below where you actually want it to paste for it to paste properly. I reverted to 2019.11 and it worked properly the first paste but then the next one, I had to paste 3 rows below. If you only select one row but multiple items, it appears to work correctly.

**Comment by AzGilrock** [comment]
You're going to need to describe this better or show a video.  I just created a lyric track and copied a section including all 3 rows of phrase, word, and phoneme's and then I pasted it in 2 additional locations so I'm not seeing the described problem.

**Comment by Poporacer** [comment]
I guess this might have been a computer problem even though I had rebooted the computer, downgraded XLights to 2019.11 and back to 2019.12 and rebooted again and the problem persisted. I even went to the Zoom room and they had me go through the same process of rebooting and the problem persisted (no one in the room had 2019.12 so they couldn't test on their system). I showed them my screen so that they could verify that I was only selecting the correct data (no accidental overlaps, cut and paste fields were the same size, etc.) and that I was performing the operation correctly and they confirmed that it wasn't. But today, when I tried to make a video of the bug, it worked.... SMH
Sorry for the inconvenience. If I can duplicate it again later, I will provide more documentation.
